### DISCUSSION TITLE
Can't download photos with usb cable during a long term time-lapse

### DISCUSSION THREAD

**Discussion by Zampa1985** [gopro/labs#1039] [Q&A]
Hi, 
20 days ago I installed my Hero 12 for a 1 year long time-lapse.
Yesterday I've been there to check the camera (it is still working good! Yeah!) and download on my Macbook the first part of the photos, but I've not been able to download them.
The software (Android file transfer for Mac) said something like: "the device is not  enabled for usb downloading".

Now I'm a little bit scared because I can't touch my GoPro and I can't remove the sd card from it because I risk to change the composition.
What can I do? 
I can't install Quik on my Macbook Air because it is too old for the app...but I'm sure I would have the same problem anyway.

Thanks for your precious help,
Andrea

**Comment by dnewman-gpsw** [comment]
No idea.  This is not a Labs error, seems weird. Determine the actual error message and find out the work arounds.  

You do have to turn the camera on for downloading, and that means pausing any capture.  Generally I use a sturdy mount, and change the SD Card, rarely any movement is added, but any slight bump can be fixed in post.
